### DISCUSSION TITLE
Storage alerts, whats using all the storage?

### DISCUSSION THREAD

**Discussion by nickchampion** [ravendb/ravendb#15864] [General]
Hi,

I have seen an alert on our live databases today that we are running low on storage, the chart on the server dashboard shows most the storage not allocated to databases, just wondering whats using this storage and whether its usable by RavenDB or whether we need to add more storage?

Whats using the 8GB in the chart below?

<img width="730" alt="image" src="https://user-images.githubusercontent.com/3949005/220582930-29eadf78-e273-44a5-a3b7-1b84b11a0477.png">

Thanks

**Comment by ayende** [comment]
Is this a cloud instance or running on your own machine?
We are reporting the the total size on the disk used.

Usually, aside from RavenDB, you also have the OS and other stuff installed as well.

**Reply by nickchampion** [reply]
This is on RavenDB Cloud, so the 10GB instance size includes all the OS too?

**Reply by ayende** [reply]
Yes, that is correct

**Comment by kamranayub** [comment]
Running into this same issue... but not sure how to fix it? If the DB is only taking 200MB, how do we free up the rest?

![image](https://user-images.githubusercontent.com/563819/230510451-bde068fa-4766-41f0-b719-a9bf31c93059.png)

**Reply by ayende** [reply]
Open a support ticket in the cloud portal, we'll look into that.
